src/webviews/apps/plus/graph/graph.scss

```diff
@@ -304,6 +304,20 @@ button:not([disabled]),
 	.codicon[class*='codicon-graph-line'] {
 		transform: translateY(1px);
 	}
+	.codicon-custom-git-branch-create {
+		&:before {
+			content: '\ea68';
+		}
+		&:after {
+			content: '\ea60';
+			position: absolute;
+			right: 0;
+			bottom: 0;
+			font-size: 0.6em;
+			line-height: normal;
+			transform: translate(-50%, 0%);
+		}
+	}
```

Comment:
This is fine for here, but a new GlIcon that we can use in other places would be nice.